packages/asset-swapper/src/utils/market_operation_utils/multihop_utils.ts

```diff
@@ -34,7 +36,7 @@ export function getIntermediateTokens(
  * Returns the best two-hop quote and the fee-adjusted rate of that quote.
  */
 export function getBestTwoHopQuote(
-    marketSideLiquidity: MarketSideLiquidity,
+    marketSideLiquidity: Omit<MarketSideLiquidity, 'makerTokenDecimals' | 'takerTokenDecimals'>,
```

Comment:
I think this was done for testing purposes. In general, bunny-hop does not care about decimals because it simply works with selling + buying base unit amount. Whatever it gets back, it then sells.